koku/api/report/queries.py

```diff
@@ -315,6 +318,23 @@ def _get_search_filter(self, filters):  # noqa C901
         LOG.debug(f"self.query_exclusions: {self.query_exclusions}")
         return composed_filters
 
+    def _provider_map_conditional_exclusions(self):
+        """
+        Uses the provider_map conditionals to exclude from a query in certain scenarios.
+
+        Such as when we fall back to the daily summary table but don't want Unallocated projects
+        included for OCP compute/memory endpoints.
+        """
+
+        exclusions = QueryFilterCollection()
+        exclude_list = (
+            self._mapper.report_type_map.get("conditionals", {}).get(self.query_table, {}).get("exclude", [])
+        )
+        if hasattr(exclude_list, "__iter__"):
```

Comment:
What purpose does this `if` statement serve? The provider-map should define the `exclude` as a list (we control this) and if it doesn't exist, then this `get` statement returns an empty list. So either way, the `exclude_list` is an iterable list.